.github/workflows/e2e.yaml

```diff
@@ -99,7 +94,7 @@ jobs:
         EXCLUDE_PROMETHEUS_ALL_NS_TESTS=${{ matrix.prometheusAllNS }}
         EXCLUDE_THANOSRULER_TESTS=${{ matrix.thanosruler }}
         EXCLUDE_OPERATOR_UPGRADE_TESTS=${{ matrix.operatorUpgrade }}
-        FEATURE_GATED_TESTS=${{ matrix.featureGated }}
+        FEATURE_GATED_TESTS=exclude
```

Comment:
should we rename to EXCLUDE_FEATURE_GATED_TESTS for consistency?

Reply:
yeah good idea